be/src/runtime/runtime_filter_cache.h

```diff
@@ -0,0 +1,62 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+#pragma once
+#include <atomic>
+#include <shared_mutex>
+#include <thread>
+#include <unordered_map>
+#include <vector>
+
+#include "exprs/vectorized/runtime_filter.h"
+#include "gen_cpp/Types_types.h" // for TUniqueId
+
+namespace starrocks {
+
+using JoinRuntimeFilterPtr = std::shared_ptr<vectorized::JoinRuntimeFilter>;
+class RfCacheValue;
+using RfCacheValueRawPtr = RfCacheValue*;
+using RfCacheValuePtr = std::shared_ptr<RfCacheValue>;
+class RfEventValue;
+using RfEventValueRawPtr = RfEventValue*;
+using RfEventValuePtr = std::shared_ptr<RfEventValue>;
+
+class RuntimeFilterCache {
+public:
+    explicit RuntimeFilterCache(size_t log2_num_slots);
+    ~RuntimeFilterCache();
+    Status init();
+    void put_if_absent(const TUniqueId& query_id, int filter_id, const JoinRuntimeFilterPtr& filter);
+    JoinRuntimeFilterPtr get(const TUniqueId& query_id, int filter_id);
+    void remove(const TUniqueId& query_id);
+    size_t cache_times() const { return _cache_times; }
+    size_t use_times() const { return _use_times; }
+    void add_rf_event(const TUniqueId& query_id, int filter_id, std::string&& msg);
+    std::unordered_map<std::string, std::list<std::string>> get_events();
+    void set_enable_trace(bool enable) {
+        _enable_trace.store(enable);
+        if (!enable) {
+            _clean_events();
+        }
+    }
+    bool enable_trace() const { return _enable_trace; }
```

Comment:
Is this variable used?

Reply:
this variables used in restful api in next pr